ansible/roles/developer_packages/tasks/main.yml

```
@@ -47,3 +47,6 @@
       - build-essential
       - vim-gui-common
       - vim-runtime
+
+  - name: install ngrok
+    shell: sudo snap install ngrok
```

I think Ansible warns about using `sudo` in `shell` though I've not done it in a while
```suggestion
    shell: snap install ngrok
    become: true
```

---

ah right- that makes sense after reading ansible docs - `become` not very intuitive though!

I used `yes` rather than `true` to follow ansible documentation